README.md

```diff
@@ -16,9 +16,10 @@ Avash provides the ability to run Lua scripts which can execute a sequence of sh
 ### Quick Setup
 
  1. Install and build an AVA client
- 2. `go get github.com/ava-labs/avash`
- 3. `cd $GOPATH/src/github.com/ava-labs/avash`
- 4. `go build`
+ 2. `cd $GOPATH`
+ 3. `go get github.com/ava-labs/avash`
+ 4. `cd src/github.com/ava-labs/avash`
```

Comment:
Why can't this just be one step like before?

`cd $GOPATH/src/github.com/ava-labs/avash`

Tada! You're in the right path.